.github/workflows/align_templates.yml

```diff
@@ -0,0 +1,28 @@
+name: Ensures template/ dir and config file are aligned.
+
+on:
+  pull_request:
+    paths:
+      - templates/**
+
+jobs:
+
+  align-dirs:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v4
+      with:
+        # Needed for HEAD^1
+        fetch-depth: 2
+
```

Comment:
Why is this necessary?

Reply:
https://github.com/SciTools/.github/pull/61/commits/8d30077e53f2d6a437f4fef4c12e216eeb781635